app/models/potential_match.rb

```diff
@@ -112,6 +118,11 @@ def self.generate_in_background(collection_id)
   # Generate potential matches for a signup in the general process
   def self.generate_for_signup(collection, signup, settings, collection_tag_sets, required_types, prompt_type = "request")
     potential_match_count = 0
+    max_matches = [
+      (collection.signups.count / ArchiveConfig.POTENTIAL_MATCHES_PERCENT),
+      ArchiveConfig.POTENTIAL_MATCHES_MAX
+    ].min
+    max_matches = [max_matches, ArchiveConfig.POTENTIAL_MATCHES_MIN].max
```

Comment:
`ArchiveConfig.POTENTIAL_MATCHES_MIN` is throwing me a little -- my first thought was "but what if there aren't ten sign-ups for the challenge, let alone ten potential matches for a given prompt?" -- but I _think_ I get it now, and I want to see if I understand what's going on here... 

So what we're doing is setting the maximum number of potential matches that may be generated for a signup, and we're deciding that by:

1. First taking the smaller of either 50 or 10% of the total signups
2. Then taking the larger number of either _that_ or 10

So in a challenge with 8 signups, it'd be like...

max_matches = [ (8/10), 50 ].min
max_matches = [ 0.8, 10 ].max

Giving us 10 as the maximum number of potential matches to generate for a signup in that challenge?